### DISCUSSION TITLE
is there a way to restrict access to a select few

### DISCUSSION THREAD

**Discussion by JunkyardCat** [jellyfin/jellyfin#10028] [Q&A] [answered]
Lets say i added a video but i just want to make it accessible to some user not the whole registered user. Is there a way to do this?

**Comment by fernvenue** [comment]
You can put the vedio to the specific library and disable access to that library from specific user.

**Reply by fernvenue** [reply, marked as answer]
Go to *Dashboard* > *Users* and choose the specific user then use *Library access* to manage the permission.

**Comment by sbsaylors** [comment]
You could have 'guest' account - and assign just a 'guest' library. That library just includes that video.

**Comment by sbsaylors** [comment]
![image](https://github.com/jellyfin/jellyfin/assets/42074466/8b71d5f2-b531-486b-9690-2c4ce61a6046)
